lib/Service/LangRopeService.php

```diff
@@ -134,15 +134,26 @@ private function requestToExApp(
 			$options,
 		);
 		if (is_array($response) && isset($response['error'])) {
-			throw new RuntimeException('Error during request to ExApp (context_chat_backend): ' . $response['error']);
+			throw new RuntimeException('Error during request to Context Chat Backend (ExApp): ' . $response['error']);
+		}
+		if (is_array($response)) {
```

Comment:
```suggestion
		if (!is_array($response)) {
```

?

Reply:
app_api only returns an array if there's an array. So if there is no "error" key in the array, this state is unexpected. We only expect a IRequest object.

Reply:
AH, makes sense